immuni_exposure_ingestion/helpers/risk_level.py

```diff
@@ -39,4 +41,5 @@ def extract_keys_with_risk_level_from_upload(upload: Upload) -> Iterable[Tempora
     for key in keys_at_risk:
         key.transmission_risk_level = TransmissionRiskLevel.highest
 
-    return keys_at_risk
+    # Also remove any keys that might still be valid
+    return [key for key in keys_at_risk if key.created_at.date() < date.today()]
```

Comment:
can you make this behaviour configurable with an env var, given that in the future it should be supported as well with different rolling periods?

Reply:
What do you mean? What exactly should be configurable?

You mean that keys should be at least X days old (where X is now 1) ?

```
[key for key in keys_at_risk if key.created_at.date() <= date.today() - timedelta(days=config.MINIMUM_AGE_OF_KEYS_IN_BATCH_IN_DAYS)]
```

Something like that?

Reply:
nope, I meant something similar to the following:

```
return [key for key in keys_at_risk if key.created_at.date() < date.today()] if config.EXCLUDE_CURRENT_DAY_TEK else keys_at_risk
```

Reply:
👍